### PR TITLE
Support schema option and add forceMigration options

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Support schema option and add forceMigration options

### Why is it needed?

Support `postgres` database schemas & disabling column & table deletion

Fixes https://github.com/strapi/strapi/issues/11367

### CODE REVIEW

**Comment by ethan-gallant** [comment]
On behalf of me, myself and I, I cannot express my excitement in words about being able to perform proper database migrations. :sweat_drops:

File: examples/getstarted/config/database.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const postgres = {
     password: 'strapi',
     port: 5432,
     host: 'localhost',
+    schema: 'coucou',
```
Comment (Elhebert) [inline examples/getstarted/config/database.js:17]: So french 🐓

**Review by Elhebert — DISMISSED** [review]
LGTM 👌

File: examples/getstarted/config/database.js (hunk 1 above)
Comment (alexandrebodin) [reply examples/getstarted/config/database.js:17]: Haha I need to remove that nice xD

**Review by Elhebert — APPROVED** [review]

**Pull request merged**